message.go

```diff
@@ -34,7 +34,7 @@ func main() {
 		return
 	}
 
-	previewURL := fmt.Sprintf("%s/#/previews/%s", oktetoURL, previewName)
+	previewURL := fmt.Sprintf("%s/previews/%s", oktetoURL, previewName)
```

Comment:
When did we change this URL (on which chart version)? are both accessible?

Reply:
Yes, [both URLs still work](https://github.com/okteto/app/blob/d0b48557bdb3402d148e406b33a8ad67c9ce7f72/frontend/src/index.html#L13-L34). The UI redirects the user to the new one if trying to access the one with the `#`. It [was changed](https://github.com/okteto/app/pull/4748) on Oct 10th 2022.